internal/config/storage.go

```diff
@@ -61,6 +63,18 @@ func (c *StorageConfig) setDefaults(v *viper.Viper) error {
 		}
 	case string(OCIStorageType):
 		v.SetDefault("store.oci.insecure", false)
+
+		configDir, err := os.UserConfigDir()
+		if err != nil {
+			return fmt.Errorf("getting user state dir: %w", err)
+		}
+
+		imageDir := filepath.Join(configDir, "flipt", "bundles")
```

Comment:
i wonder if we should have a helper func that we use to get the `flipt` state dir, since we do it here now and i think in a couple other places

Reply:
I just pushed a commit for this. See what you think.